WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1224,23 +1224,32 @@ extension AztecPostViewController : Aztec.FormatBarDelegate {
         updateFormatBar()
     }
 
+    func formatBar(_ formatBar: FormatBar, didChangeOverflowState overflowState: FormatBarOverflowState) {
+        let action = overflowState == .visible ? "made_visible" : "made_hidden"
+        WPAppAnalytics.track(.editorTappedMoreItems, withProperties:[WPAppAnalyticsKeyEditorSource: Analytics.editorSource, "action": action], with: post)
+    }
+
 
     func toggleBold() {
+        WPAppAnalytics.track(.editorTappedBold, withProperties:[WPAppAnalyticsKeyEditorSource: Analytics.editorSource], with: post)
```

Comment:
Should we have a single method within `AztecPostViewController` that can just track an event and send the right properties? So these calls would then just become `track(.editorTappedBold)` for example.